File: dry_cleaning_system_HmOS/entry/src/main/ets/models/Customer.ts
/**
 * 客户数据模型
 */
export interface Customer {
  id: number;
  name: string;
  phone: string;
  wechat: string;
  balance: number;
  createTime: string;
}

/**
 * 创建客户请求
 */
export interface CreateCustomerRequest {
  name: string;
  phone: string;
  wechat?: string;
  balance?: number;
}

/**
 * 更新客户请求
 */
export interface UpdateCustomerRequest {
  name?: string;
  phone?: string;
  wechat?: string;
  balance?: number;
}

File: dry_cleaning_system_Android/app/src/main/java/com/chaser/drycleaningsystem/data/entity/Customer.kt
package com.chaser.drycleaningsystem.data.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * 客户实体类
 */
@Entity(tableName = "customer")
data class Customer(
    @PrimaryKey(autoGenerate = true) val id: Long = 0,
    @ColumnInfo(name = "name") val name: String,
    @ColumnInfo(name = "phone") val phone: String?,
    @ColumnInfo(name = "wechat") val wechat: String?,
    @ColumnInfo(name = "balance") val balance: Double = 0.0,
    @ColumnInfo(name = "create_time") val createTime: String = System.currentTimeMillis().toString(),
    @ColumnInfo(name = "note") val note: String? = null
)
